core/src/main/java/org/testng/internal/MethodHelper.java

```diff
@@ -456,6 +459,13 @@ protected static String calculateMethodCanonicalName(Class<?> methodClass, Strin
         .orElse(null);
   }
 
+  public static void clear(List<Method> methods) {
+    Optional.ofNullable(methods)

```

Comment:
List to Stream conversion

I think you can directly use Stream and avoid the intermediate List

Reply:
fixed